pkg/processor/sumologicschemaprocessor/README.md

```diff
@@ -67,6 +67,10 @@ processors:
       - attribute: <attribute>
         prefixes: [<prefix>]
       - ...
+
+    # Defines whether `severity` attribute should be added to record attributes.
+    # default = true
+    add_severity_level_attribute: {true, false}
```

Comment:
I propose the following configuration, or something similar:

```yaml
add_severity_attribute:
  enabled: true
  attribute_name: loglevel
```

Reply:
Changed configuration and extended to use other fields as well, eg. `severity text` `span id` and `trace id`. That was asked by our customer support, however I'm still waiting for confirmation if client needs that. Anyway, we can always leave it or remove.

Reply:
shouldn't this be updated?

Reply:
@sumo-drosiek what do you mean? merge/rebase with main?

Reply:
I agree with @astencel-sumo that the configuration added by this PR should be grouped in one major key, instead of producing 4 new config options

Reply:
I missed this one

Reply:
@sumo-drosiek I don't think @astencel-sumo suggested grouping, but maybe I'm wrong?

Reply:
@astencel-sumo? 🤔

Reply:
Hi @pdelewski any update on this? The customer is inquiring for updates and I don't see any recent movement. Thank you.

Reply:
Hi @bnartiff , I will update PR tomorrow. It still requires aprovals. I was also waiting for confirmation regarding fields that customer expects (as originally there were mentioned only severity, recently I updated that to `spanid` and `traceid`

Reply:
Thanks for the update, I am going to check w/ the customer to confirm the desired fields. Thank you for adding spanid and traceid as well.

Reply:
Per customer, "I think severity, spanID, and traceID are a good start.  I don’t have any other suggestions/needs at this point. " @pdelewski